modules/nf-core/eklipse/main.nf

```diff
@@ -0,0 +1,56 @@
+
+process EKLIPSE {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "bioconda::eklipse=1.8"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/eklipse:1.8--hdfd78af_1':
+        'biocontainers/eklipse:1.8--hdfd78af_1' }"
+
+    input:
+    tuple val(meta), path(bam)
+    path ref_gb
+
+    output:
+    tuple val(meta), path("*deletions.csv"), path("*genes.csv"), emit: csv
+    tuple val(meta), path("*.png")                             , emit: circos
+    path "versions.yml"                                        , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def ref_gb = ref_gb ? "$ref_gb" : "/usr/local/bin/data/NC_012920.1.gb"
```

Comment:
It will work when the input data is human, as it is mt data, it will be uncommon to use another reference. However, if you want to, you can always give it a new reference using the ref_gb input

Reply:
Allright perfect! It won't work in a conda environment though, but that's not that big of a problem